sky/packages/sky/lib/src/widgets/page.dart

```diff
@@ -0,0 +1,137 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/animation.dart';
+
+import 'basic.dart';
+import 'framework.dart';
+import 'navigator2.dart';
+import 'overlay.dart';
+import 'transitions.dart';
+
+abstract class TransitionRoute extends Route {
```

Comment:
TransitionRoute will be used by DialogRoute, which isn't a Page. Should TransitionRoute be in a different file, maybe the same file as Route?